src/main/java/com/example/messageserver/service/PriceService.java

```diff
@@ -31,15 +32,16 @@ public class PriceService {
 
     private final ModelMapper modelMapper;
 
-    public void calculatePrice(BatteryStatisticDto batteries) {
+
+    private final String baseUrl = "https://813df768-ced3-44c0-a14c-73c15ada1666.mock.pstmn.io";
```

Comment:
Constant names are usually named by convention that includes all uppercase names and using snake notation (BASE_URL).

Reply:
Also, this concrete value should be loaded from application.yaml.